sql/moz-fx-data-shared-prod/telemetry_derived/events_last_seen_v1/query.sql

```diff
@@ -0,0 +1,59 @@
+WITH _current AS (
+  SELECT
+    submission_date,
+    sample_id,
+    client_id,
+    -- In this raw table, we capture the history of activity over the past
+    -- 28 days for each usage criterion as a single 64-bit integer. The
+    -- rightmost bit represents whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_logged_event_bits,
+    CAST(
+      LOGICAL_OR(event_method = 'show' AND event_object = 'protection_report') AS INT64
+    ) AS days_viewed_protection_report_bits,
+    CAST(
+      LOGICAL_OR(event_category = 'pictureinpicture' AND event_method = 'create') AS INT64
+    ) AS days_used_pip_bits,
```

Comment:
reading through this, I was not able to figure out what `pip` meant without looking for where this field is defined. it may be worth spelling it out. Then again maybe not, that was very difficult to type correctly just now. If `pip` is already being used elsewhere to mean picture in picture, then ignore this.
```suggestion
      LOGICAL_OR(event_category = 'pictureinpicture' AND event_method = 'create') AS INT64
    ) AS days_used_picture_in_picture_bits,
```